Autonomous decision making demonstration.ipynb

```diff
@@ -126,22 +126,22 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 14,
    "metadata": {},
    "outputs": [],
    "source": [
     "# Defining obstacles notation x position y position size\n",
-    "obstacles = [(20,20,5,5),(20,40,5,5),(40,20,10,10)]\n"
+    "obstacles = [(100,20,20,5),(100,20,40,5),(200,40,20,10)]\n"
```

Comment:
Why are you changing the weightings on things? What effect does this have?

Reply:
I changed the obstacles to be higher cost so the path avoids them